core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadStatus.java

```diff
@@ -109,18 +95,41 @@ public static DownloadStatus fromCursor(Cursor cursor) {
 		int indexCompletionDate = cursor.getColumnIndex(PodDBAdapter.KEY_COMPLETION_DATE);
 		int indexReasonDetailed = cursor.getColumnIndex(PodDBAdapter.KEY_REASON_DETAILED);
 
-		long id = cursor.getLong(indexId);
-		String title = cursor.getString(indexTitle);
-		long feedfileId = cursor.getLong(indexFeedFile);
-		int feedfileType = cursor.getInt(indexFileFileType);
-		boolean successful = cursor.getInt(indexSuccessful) > 0;
-		int reason = cursor.getInt(indexReason);
-		Date completionDate = new Date(cursor.getLong(indexCompletionDate));
-		String reasonDetailed = cursor.getString(indexReasonDetailed);
-
-		return new DownloadStatus(id, title, feedfileId,
-				feedfileType, successful, DownloadError.fromCode(reason), completionDate,
-				reasonDetailed);
+		return new DownloadStatus(cursor.getLong(indexId),
+								  cursor.getString(indexTitle),
+								  cursor.getLong(indexFeedFile),
+								  cursor.getInt(indexFileFileType),
+								  cursor.getInt(indexSuccessful) > 0,
+								  false,
+								  true,
+								  DownloadError.fromCode(cursor.getInt(indexReason)),
+								  new Date(cursor.getLong(indexCompletionDate)),
+								  cursor.getString(indexReasonDetailed),
+								  false);
```

Comment:
This file seems to be indented with tabs instead of spaces. While you are already touching big parts of the file, it would make sense to let Android Studio convert the file to spaces instead of tabs. That would also fix the strange indention above.

Reply:
Changed.